spec/services/proforma_service/export_tasks_spec.rb

```diff
@@ -32,26 +32,26 @@
     end
     let(:doc) { Nokogiri::XML(zip_files['task.xml'], &:noblanks) }
     let(:xml) { doc.remove_namespaces! }
-    let(:imported_exercises) { zip_files.transform_values! { |zip_file| ProformaService::Import.call(zip: zip_file, user: user) } }
+    # let(:imported_tasks) { zip_files.transform_values! { |zip_file| ProformaService::Import.call(zip: zip_file, user: user) } }
 
     it 'creates a zip-file with two files' do
       expect(zip_files.count).to be 2
     end
 
-    it 'creates a zip-file of two importable zip-files' do
-      expect(imported_exercises.values).to all be_an(Exercise)
+    xit 'creates a zip-file of two importable zip-files' do
```

Comment:
We are currently not supporting the export of two exercises at the same time (the former cart), I assume?

Reply:
The service actually does already support exporting multiple tasks. 
The problem here is, that the specs uses the importer, which is not yet migrated to use `tasks` instead of `exercises` (#505)

Reply:
Ah, I see. This is really complex and hard for me to consider everything. If you'd like, we can also have a chat on (this /) the other MR ;).